detectors/node/opentelemetry-resource-detector-instana/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenTelemetry Resource Detector for Instana
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+This resource detector will detect the Instana agent to register the Opentelemetry as a Node.js process. The created resource will be automatically merged with the existing resources and contains the real PID, which is returned from the Instana agent. This mechanism is needed to be able to display the Opentelemetry process on the host in the Instana UI and to connect the Node.js process with the incoming Opentelemetry spans.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/resource-detector-instana
+```
+
+## Environment variables
+
+- INSTANA_AGENT_HOST: The Instana agent hostname.
+- INSTANA_AGENT_PORT: The Instana agent port.
+- INSTANA_RETRY_TIMEOUT: The resource detector does three retries to connect to the Instana agent. This is the timeout between the retries.
+- INSTANA_AGENT_TIMEOUT: The client timeout when connecting the Instana agent.
+
+## Usage
+
+```typescript
+import {
+  Resource,
+  processDetector,
+  envDetector,
+} from "@opentelemetry/resources";
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { instanaAgentDetector } from "@opentelemetry/resource-detector-instana";
+
+const globalResource = new Resource({
+   [SemanticResourceAttributes.SERVICE_NAME]: "TestService",
+});
+
+const sdk = new NodeSDK({
+   autoDetectResources: false,
+   resource: globalResource,
+});
+
+await sdk.detectResources({
+   detectors: [envDetector, processDetector, instanaAgentDetector],
+});
+
+// console.log(sdk['_resource'])
```

Comment:
I don't think we should suggest users to access the resource like this

Reply:
Its only a hint for debugging purposes. Can be removed, I dont mind

Reply:
Removed